tests/performance/plugins/server-timing/mu-plugin.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+add_action(
+	'template_redirect',
+	function() {
+
+		global $server_timing_values, $timestart, $template_start;
```

Comment:
Nit-pick: Better not to make `$template_start` a global variable, as it's only needed in here and the closure below. You can pass it to the other closure via `use()`.

Maybe we should even keep `$server_timing_values` as just a local variable for now. At least for now, the way this code is written there is no real benefit in having it as a global. That would change if we want to add server timing values from elsewhere, but at least now we don't, so maybe best to avoid the global.

Reply:
Thanks @felixarntz Good catch.